front-end/src/types/Post.ts

```diff
@@ -0,0 +1,19 @@
+export interface Post {
+  _id: string
+  _type: string
+  title: string
+  description: string
+  source: string
+  origin: string
+  visibility: string
+  unlisted: boolean
+  isPrivateToFriends: boolean
```

Comment:
TODO: probably gonna remove this when we fix up on the models in the backend since I accidentally left it in

Reply:
@VinnyLuu Could you remove that from the backend in a PR now so Chris can just remove isPrivateToFriends?

Reply:
Sure I'll get it up rn